index.bs

```diff
@@ -648,23 +665,18 @@ ReadableStream(<var>underlyingSource</var> = {}, { <var>size</var>, <var>highWat
        initiate further reads from _reader_ or writes to _writer_. (Ongoing reads and writes may finish.) In particular,
        the user agent must check the below conditions on *this*.[[state]] and _dest_.[[state]] before performing any
        reads or writes, since they might lead to immediate shutdown.
```

Comment:
Maybe the PR could simply add "in order" after "before performing any reads or writes"?